src/main/webapp/src/app/find-gardens-page/find-gardens.component.ts

```diff
@@ -71,11 +73,27 @@ export class FindGardensComponent implements OnInit {
    *
    * @param route Contains arguments.
    */
-  constructor(private route: ActivatedRoute, private httpClient: HttpClient) {}
+  constructor(private route: ActivatedRoute, private httpClient: HttpClient) {
+    // Add simple regex zip code validation.
+    this.zipCodeControl = new FormControl('', [
+      Validators.pattern('^$|^([0-9]{5})$'),
+    ]);
```

Comment:
regex! This is just to confirm the zipcode is only numbers and 5 digits long, correct?

Reply:
The empty string is also accepted by this regex -- this just searches for gardens near the user.